core/chain_util.go

```diff
@@ -32,7 +32,7 @@ import (
 var (
 	blockHashPre  = []byte("block-hash-")
 	blockNumPre   = []byte("block-num-")
-	expDiffPeriod = big.NewInt(100000)
+	ExpDiffPeriod = big.NewInt(100000)
```

Comment:
Right, my question is how it's not applied - looking at the code it seems this will be used for non-Frontier nets?